onebusaway-android/src/main/java/org/onebusaway/android/nav/NavigationService.java

```diff
@@ -180,7 +180,8 @@ public synchronized void onLocationChanged(Location location) {
         } else if (!LocationUtils.isDuplicate(mLastLocation, location)) {
             mNavProvider.locationUpdated(location);
         }
-        if (BuildConfig.NAV_GPS_LOGGING) {
+
+        if ((BuildConfig.NAV_GPS_LOGGING) &&(mNavProvider.sectoCurDistance <= 300)) {
```

Comment:
Instead of hardcoding 300 here, could you reference the variable defined elsewhere (I think) for the Get Ready notification?

Reply:
Thank you for the comment!
The variable 'DISTANCE_THRESHOLD' used for Get Ready notification has a value of 200. Can you please let me know whether it is ok to replace the hardcoded value to DISTANCE_THRESHOLD + 100 or a new variable should be created for the trip log threshold?

Reply:
Let's add a new variable `RECORDING_THRESHOLD = DISTANCE_THRESHOLD + 100` - this seems reasonable to me.  We want the recording to start before the Get Ready notification, so having a hard coded value plus the radius for Get Ready makes sense, and having this in a descriptive constant makes it easier to read the code.

Reply:
Thank you for the reply! I have added the new variable to the code.